notebooks/book2/10/invere_cdf_sampling.ipynb

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    plt.scatter(0, uniform_samples[0], s=POINT_SIZE, c="r", label=r"samples $U(0,1)$")
</pre><p>Unif(0, 1) is preferred in the book instead of U(0,1)</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/60/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>